src/applications/ivc-champva/10-10D/config/form.js

```diff
@@ -185,6 +185,7 @@ const formConfig = {
       title: 'Signer information',
       pages: {
         page2: {
+          // initialData: mockData.data,
```

Comment:
Intentionally added. Makes testing easier when I can just comment/uncomment this line as needed.